pkg/crd/crd.go

```diff
@@ -85,16 +161,16 @@ func EnsureFissionCRDs(logger *zap.Logger, clientset *apiextensionsclient.Client
 				Name: "environments.fission.io",
 			},
 			Spec: apiextensionsv1beta1.CustomResourceDefinitionSpec{
-				Group:   crdGroupName,
-				Version: crdVersion,
-				Scope:   apiextensionsv1beta1.NamespaceScoped,
+				Group: crdGroupName,
+				Scope: apiextensionsv1beta1.NamespaceScoped,
 				Names: apiextensionsv1beta1.CustomResourceDefinitionNames{
 					Kind:     "Environment",
 					Plural:   "environments",
 					Singular: "environment",
 				},
 				PreserveUnknownFields: boolPtr(false),
-				Validation:            environmentValidation,
```

Comment:
Is there a reason we are removing the validation here and other places?

Reply:
They've been moved to the upper layer in the `Schema`.   https://github.com/fission/fission/pull/1938/files#diff-1d74bd0caacd00451f3f6d75e70ac4a780cfe74a82fb5fa774e464935e68680fR80